packages/terser/src/module.ts

```diff
@@ -1,13 +1,17 @@
+import { fileURLToPath } from 'url';
+
 import type { NormalizedOutputOptions, RenderedChunk } from 'rollup';
 import { hasOwnProperty, isObject, merge } from 'smob';
 
 import type { Options } from './type';
 import { WorkerPool } from './worker-pool';
 
-export default function terser(options: Options = {}) {
+export default function terser(input: Options = {}) {
+  const { maxWorkers, ...options } = input;
```

Comment:
I believe the destructuring for this could happen in the same place the parameter is defined.

```suggestion
export default function terser({ maxWorkers, ...options }: Options = {}) {
```